src/lib.rs

```diff
@@ -1,4 +1,5 @@
-use red4ext_rs::{prelude::*, interop::TweakDBID};
+use red4ext_rs::{prelude::*};
+use red4ext_sys::interop::TweakDBID;
```

Comment:
You should re-export the type from the main crate by adding a `pub use` item in the types.rs file. That way users don't need to depend on the sys crate.